src/DigitalSubscriberAppBanner/index.test.tsx

```diff
@@ -22,19 +21,15 @@ describe('DigitalSubscriberAppBanner', () => {
             const { getByText } = render(
                 <DigitalSubscriberAppBanner
                     {...baseProps()}
-                    submitComponentEvent={logButtonClickWithOphan}
+                    trackClick={logButtonClickWithOphan}
```

Comment:
Nitpick: shall we rename `logButtonClickWithOphan` to reflect the broader use of this callback now (i.e. it doesn't just log to Ophan).

Reply:
Renamed to `logButtonClick` across relevant banner test files which mock such a function